python/lsst/sitcom/vandv/mount.py

```diff
@@ -46,3 +47,270 @@ async def moveMountInElevationSteps(
         el = target_el
 
     return azimuth, el
+
+def get_slew_pairs(starts, stops, return_unmatched=False):
+    """
+    Given vectors of start times and stop times take the longer vector
+    and iterate over it. If that is starts, for each start time select all stop
```

Comment:
Can you use backticks when referring to variables? I believe it would make the docstring a bit clearer.